app/views/posts/_new_offer.html.erb

```diff
@@ -1,15 +1,9 @@
-<div class="container">
```

Comment:
I think every page still needs to have <div class="container mt-3"> though in order for our styling to be consistent

Reply:
disregard, i see your work further below. great 👍